nuxeo-java-client/src/main/java/org/nuxeo/client/objects/directory/DirectoryEntry.java

```diff
@@ -64,22 +66,33 @@
 
     protected Map<String, Object> properties = new HashMap<>();
 
+    /**
+     * Regular way to instantiate a {@link DirectoryEntry} in order to create it.
+     */
     public DirectoryEntry() {
         super(EntityTypes.DIRECTORY_ENTRY, DirectoryManagerAPI.class);
     }
 
+    /**
+     * Regular way to instantiate a {@link DirectoryEntry} in order to update it.
+     *
+     * @since 3.12.0
+     */
+    public DirectoryEntry(String id) {
```

Comment:
it seems this is the new signature you added. Shouldn't the `* @since 3.12.0` stay on `public DirectoryEntry()` ? And then this one would have a new @since ?

Reply:
I put the since on the newly created API, which is `DirectoryEntry(String)`, the other one without parameter was already there, and didn't have _since_ since it is there from the creation of the class.

I may not have got your question.

Reply:
No, I forgot this is java client repo 3.X seemed super old to me :)